### ISSUE TITLE
web-wallet: Add Build Step for w3sper SDK to Enable Web Wallet Integration

### ISSUE THREAD

**Issue by nortonandreev** [dusk-network/rusk#2612]
### Description:

We are currently migrating the service used by our SvelteKit Web Wallet application (located in the `web-wallet` folder of our mono-repo) to use the new `w3sper` JavaScript SDK. This SDK is housed within the `w3sper` folder of the same monorepo and is a Deno-based application.

Since the `w3sper` SDK is in its pre-alpha stage and not yet available in the npm registry, the Web Wallet cannot directly reference it using standard npm installation methods. Additionally, the `w3sper` SDK does not yet include its own build script, which is required for integration into the Web Wallet project.

### Current Roadblock:
To facilitate quicker development and testing, we need the `w3sper` SDK to generate a build output that can be referenced by the Web Wallet application using the `file://` protocol. However, as of now:
- The SDK does not have an existing build step.
- The Web Wallet is unable to integrate the SDK directly due to the absence of a build artifact.

### Proposed Solution:

1. **Add Build Script for w3sper SDK:**
    - Implement a simple build process for the `w3sper` SDK. The goal is to compile the SDK code into a distributable format (e.g., dist folder inside the w3sper directory).
    - Ensure that the output files are in a format that the Web Wallet can reference using the `file://` protocol.
    - Update the build step so that it is easily runnable, either through a dedicated npm script or a Deno-based build tool.

2. **Integration with Web Wallet:**
    - Modify the Web Wallet build configuration to reference the SDK via the file:// protocol from the w3sper/dist folder.
    - Ensure the Web Wallet can consume the build artifact during development and testing, allowing us to progress with the migration despite the SDK’s pre-alpha status.

### Tasks:

1. Add a basic build script for the w3sper SDK to output the compiled files in the dist folder.
2. Configure the Web Wallet to reference the built SDK from the local file system.
3. Test the Web Wallet integration with the locally built w3sper SDK to ensure smooth migration.
4. Document the build and integration steps within the monorepo’s README to guide developers through the process.

### Additional Context:
The migration to the new w3sper SDK is critical for future features and integrations, so having this build step in place will unblock development efforts.

**Comment by nortonandreev** [comment]
Pausing work on the build script for now. After discussing with Matteo, we’ve agreed to include w3sper in our folders as is and merge directly to master. This approach will simplify things by avoiding the extra step of an integration branch.

cc: @ascartabelli

**Comment by nortonandreev** [comment]
No longer needed, w3sper is released on [jsr](http://jsr.io/@dusk/w3sper).
